server/controllers/ItemMockCtrl.js

```diff
@@ -0,0 +1,95 @@
+const R = require('ramda');
+
+const ItemMockCtrl = (() => {
+  const categories = ['A', 'B', 'C', 'D'];
+  const validateCategoryOrNil = (value) => {
+    return R.or(R.isNil(value), R.includes(value, categories));
+  };
+
+  const validateStringOrNil = (value) => {
+    return R.or(R.isNil(value), R.is(String, value));
+  };
+  const findName = (name1, arr) => R.compose(
+    (value) => R.includes(value, name1),
```

Comment:
Pour les paramètres ce serait plutôt l'inverse : 
```suggestion
    (value) => R.includes(name1, value),
```

Qu'on peut simplifier : 
```suggestion
    R.includes(name1),
```